otx/algorithms/classification/configs/mobilenet_v3_large_1_cls_incr/semisl/template_experimental.yaml

```diff
@@ -0,0 +1,60 @@
+# Description.
+model_template_id: Custom_Image_Classification_MobileNet-V3-large-1x
+name: MobileNet-V3-large-1x
+task_type: CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Semi-Supervised Image Classification for MobileNet-V3-large-1x
+application: ~
+
+# Algo backend.
+framework: OTEClassification v1.2.3
+
+# Task implementations.
+entrypoints:
+  base: otx.algorithms.classification.tasks.ClassificationTrainTask
+  openvino: otx.algorithms.classification.tasks.ClassificationOpenVINOTask
+  nncf: otx.algorithms.classification.tasks.nncf.OTXClassificationNNCFTask
```

Comment:
Do we have a working nncf implementation for the semi-sl right now? Will optimize work with this template if we omit unlabeled data?

Reply:
It should be discussed for all semi-sl tasks. @supersoob @jaegukhyun  Thank you for opinion.